buildlib/azure-pipelines-release-drp.yml

```diff
@@ -6,9 +6,17 @@ pr:
   - master
   - v*.*.x
 
+schedules:
+- cron: '0 0 * * *'
```

Comment:
we probably should not run it nightly, once a week i smore then enough

Reply:
I'd better check it daily to catch it faster and make it easier to find the root cause.

Reply:
This is way too often, and since these processes are sensitive to false positive failures i dont want you (or me if you are not here) to have to monitor and debug on daily basis.
We also need to talk about who/what is monitoring this process this i snot something that only you should be aware of.

Reply:
Ok